src/dune_rules/coq/coq_rules.ml

```diff
@@ -404,20 +443,14 @@ let setup_coqdep_for_theory_rule
        the more conservative path and pass -boot, we don't care here
        about -noinit as coqdep ignores it *)
     match coq_modules with
-    | [] -> Memo.return `Bootstrap_prelude
-    | m :: _ -> boot_type ~dir ~use_stdlib ~wrapper_name ~coq_lang_version m
+    | [] -> Bootstrap.empty
```

Comment:
@Alizter that's the point where we need `Bootstrap.empty`, indeed that's a hack, so we should maybe refactor this at some point to emit not rules when there are no coq modules?